openfga/tests.fga.yaml

```diff
@@ -27,6 +27,40 @@ tuples:
       relation: assignee
       object: role:ro-role-2
 
+    - user: role:ro-role-2#assignee
+      relation: administrator
+      object: controller:ro-controller-1
+    - user: role:ro-role-2#assignee
+      relation: audit_log_viewer
+      object: controller:ro-controller-2
+
+    - user: role:ro-role-2#assignee
+      relation: administrator
+      object: model:ro-model-1
```

Comment:
weren't we just complaining about confusing terminology? as i read these test i'm super confused by prefixes such as `user:gr-user-1` or `model:ro-model-1`..  i'd rather see we just used regular names, not names that "indicate" the purpose of the thing being named..

Reply:
I agree, but I followed the convention (and I get why piotr did it, to avoid collisions)